app/src/main/java/io/pslab/fragment/AboutUsFragment.java

```diff
@@ -43,6 +44,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View aboutPage = new AboutPage(getActivity())
                 .isRTL(false)
                 .setImage(R.drawable.logo200x200)
+                .addItem(new Element(getString(R.string.version) + "   " + getString(R.string.flavor), R.drawable.icon_version))
```

Comment:
Better add it as a new item. 
If it's PlayStore; > PlayStore version
If it's Fdroid; > Fdroid version

Reply:
Use <i class="material-icons">widgets</i> icon